### ISSUE TITLE
Write Tests and set up Travis.CI

### ISSUE THREAD

**Issue by dmadisetti** [AnotherGroupChat/maeve#23]
https://doc.rust-lang.org/1.5.0/book/testing.html

https://travis-ci.org/

**Comment by pbateman828** [comment]
throw this into an evaluate.rs file

let tokens = tokenize(&src.prompt()); // testing to see if token does what it's supposed to.
            for val in tokens {
                src.print(&format!("Iterator {}", val));
            }


Also

let tokens = tokenize(&src.prompt());

error[E0597]: borrowed value does not live long enough
  --> src/main.rs:52:54
   |
52 |             let test_tokens = tokenize(&src.prompt()); // testing to see if token does what it's supposed to.
   |                                         ------------ ^ temporary value dropped here while still borrowed
   |                                         |
   |                                         temporary value created here
...
56 |         }
   |         - temporary value needs to live until here
   |
   = note: consider using a `let` binding to increase its lifetime

idk why

**Comment by dmadisetti** [comment]
Remove the ref, and change `tokenize(text: &str)` -> `tokenize(text: &String)`?

**Comment by pbateman828** [comment]
Tried it like you said. Same thing. I'll look into to it tomorrow.
Also travis is saying, "You don't have sufficient rights to enable this repo on Travis. Please contact the admin to enable it or to receive admin rights yourself."

**Comment by pbateman828** [comment]
I looked into it and came up with the same thing as before.

I created a branch with three different tests that attempt to accomplish the same thing. 
[https://github.com/AnotherGroupChat/maeve/blob/eval/src/interpreter/token.rs](url)

For Rust, usually unit tests are thrown at the bottom of whichever file the tests are being performed. This sounds better from an organization stand point, we can just look at the bottom of the file for tests.
